// --------------------------------------------------------
// WARNING: XVIZ Protobuf files are unstable, do not use
// --------------------------------------------------------

syntax = "proto3";

package xviz.v2;

option csharp_namespace = "xviz.V2";
option go_package = "v2pb";
option java_multiple_files = true;
option java_outer_classname = "OptionsProto";
option java_package = "com.xviz.v2";
option objc_class_prefix = "XVIZ";
option php_namespace = "Xviz\\V2";

import "google/protobuf/descriptor.proto";

extend google.protobuf.MessageOptions {
  string xviz_json_schema = 54200;
}
